src/ConfigFiles/nativeLang.xml

```diff
@@ -22,8 +22,7 @@
 
 <GUP_NativeLangue name = "English">
 	<PopupMessages>
-		<MSGID_NOUPDATE>No update is available.</MSGID_NOUPDATE>
-		<MSGID_NOUPDATE2>The auto-update may not have been activated yet.</MSGID_NOUPDATE2>
+		<MSGID_NOUPDATE>No update is available.\r\rThe auto-update may not have been activated yet.</MSGID_NOUPDATE>

```

Comment:
Sorry for my last error comment:
```
<MSGID_NOUPDATE>No update is available.

The auto-update may not have been activated yet.</MSGID_NOUPDATE>
```

It should be:
`<MSGID_NOUPDATE>No update is available.&#xD;&#xD;The auto-update may not have been activated yet.</MSGID_NOUPDATE>`

`&#xD;` will be interpreted as the carriage return correctly. So we don't need `unescape()` function.

Please replace `\r` by `&#xD;` in `<MSGID_CLOSEAPP>` tag as well.

Reply:
Done.
Thank you. That was new to me.